### ISSUE TITLE
Unzip not installed on armhf image ?

### ISSUE THREAD

**Issue by rfenouil** [haugene/docker-transmission-openvpn#745]
First, thank you for the great work.

I'm trying to run your docker image from a 'raspberry-like' arm device under OMV.

In order to benefit from the recent update you made (tinyproxy config file path, and nordvpn recommended server script), I tried to pull and start the dev version but I'm getting the following error from the logs:

2019-03-21 14:07:27 [41mERROR:[0m This script requires unzip but it's not installed. Please install it and run again.

I am quite surprised to see that because 'unzip' seems to be one of the first things installed in the dockerfile... Am I missing something ?

Thank you for your help.

Image ID: cac9e041e1933a28baec4b988cc2e8651b1d668221881d5c191d05d445746b5d

My Docker version:

Server: Docker Engine - Community
 Engine:
  Version:          18.09.3
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.8
  Git commit:       774a1f4
  Built:            Thu Feb 28 06:00:18 2019
  OS/Arch:          linux/arm64
  Experimental:     false

**Comment by rfenouil** [comment]
Forgot to add that this error does not show up on 'latest' tagged image.

**Comment by rfenouil** [comment]
After looking closer to the Dockerfile, it appears that unzip is removed just after being installed using the following command:

```    && apt-get purge git unzip \``` (line 22)

Is that required for something else ?


Another question after that:
port 8888 does not seem to be exposed in armhf version, and I could not find nordvpn specific environment variables that appeared in main Dockerfile (previous merge). Is this something to be added later or is it kept in current state on purpose ?

I hope I could help better than just asking questions but I'm not experienced enough to figure out solutions by myself yet. Anyway I'll experience a bit more and come back with comments or more info.

See you later and thank you for your help.

**Comment by haugene** [comment]
Aha, look at that. Then I guess it was installed for a specific task in the Dockerfile and removed after to reduce the image footprint. But these are tools that are handy to keep in the image. Feel free to remove it and submit a PR. I might have a look at it later, but have to run soon and it's probably not before next weekend I guess.

The Nordvpn variables should also exist in the other versions. That's just an oversight. That being said I also think the image will be a bit messy if we'll have too many provider specific variables like that. So we'll need to find a solution to that as we try to make more providers "auto updated" by scripts. Either at build or run-time.

**Comment by sm0ke83** [comment]
Ran in to this, no backup... I searched, but couldn't find solution to revert back to older version on armhf. No tag for that, it's not possible, correct?
If so, only thing I can try (tomorrow) is setting provider variable to ovpn and openvpn_config to specify ovpn manually i guess? Using NordVPN. Or use openvpn in terminal and not use docker for now...

About NordVPN, saw the script needs to be fixed (no default.ovpn). Is default.ovpn meant te be referred to / speciefied manually?

New to this stuff, might be a stupid question...
Thanks in advance!
